src/SFA.DAS.EAS.Account.Api.Types/LegalEntityViewModel.cs

```diff
@@ -0,0 +1,16 @@
+using System;
+using System.Collections.Generic;
+
+namespace SFA.DAS.EAS.Account.Api.Types
+{
+    public class LegalEntityViewModel : IAccountResource
+    {
+        public long Id { get; set; }
+        public string RegisteredAddress { get; set; }
+        public string Source { get; set; }
+        public string CompanyStatus { get; set; }
+        public string Name { get; set; }
+        public string Code { get; set; }
+        public DateTime? DateOfIncorporation { get; set; }
```

Comment:
At the moment we only get this from companies house however in future we cannot rule out that we may get these date for other types of organisations later on. Given not organisations are companies it would be misleading to return a generic legal entity with a non generic term given this is an API and may be consumed by others later on.